lib/gpio.ex

```diff
@@ -40,6 +40,12 @@ defmodule Gpio do
     GenServer.call pid, {:write, value}
   end
 
+  def write(pid, value) when is_boolean(value) do
```

Comment:
What do you think of coding this as:
```
def write(pid, true), do: write(pid, 1)
def write(pid, false), do: write(pid, 0)
```
and updating the comment above that values can be true or false as well? BTW, I'm surprised this hasn't come up before. It seems so convenient to pass true and false.